modules/odc-interacting-with-applications-and-components.adoc

```diff
@@ -18,3 +18,13 @@ This feature is available only when you create applications using the *From Git*
 * Hover your cursor over the lower left icon on the Pod to see the name of the latest build and its status. The status of the application build is indicated as *New* (image:odc_build_new.png[title="New Build"]), *Pending* (image:odc_build_pending.png[title="Pending Build"]), *Running* (image:odc_build_running.png[title="Running Build"]), *Completed* (image:odc_build_completed.png[title="Completed Build"]), *Failed* (image:odc_build_failed.png[title="Failed Build"]), and *Canceled* (image:odc_build_canceled.png[title="Canceled Build"]).
 * Use the *Shortcuts* menu listed on the upper-right of the screen to navigate components in the *Topology* view.
 * Use the *List View* icon to see a list of all your applications and use the *Topology View* icon to switch back to the *Topology* view.
+* Use the *Display Options* drop-down list to configure the Topology view of the various application groupings. The options are available depending on the types of components deployed in the project:
+** *Pod Count*: Select to show the number of pods of a component in the component icon.
+** *Event Sources*: Toggle to show or hide the event sources.
+** *Virtual Machines*: Toggle to show or hide the virtual machines.
+** *Labels*: Toggle to show or hide the component labels.
+** *Application Groupings*: Clear to condense the application groups into cards with an overview of an application group and alerts.
+** *Helm Releases*: Clear to condense the components deployed as Helm Release into cards with an overview of a given release.
+** *Knative Services*: Clear to condense the Knative Service components into cards with an overview of a given component.
+** *Operator Groupings* Clear to condense the components deployed with an Operator into cards with an overview of the given group.
+* Use the field *Find by name* to highlight the component names that match the entry. Press kbd:[Enter]. Click the information icon (image:odc_info.png[title="Information"]) to resize the topology view to fit the search results outside of the current visible space.
```

Comment:
Highlight components you want to see by entering the component name in the *Find by name* field and click the information icon (image:odc_info.png[title="Information"]) to resize the topology view to fit the search results outside of the current visible space.

Reply:
* Use the *Find by name* field to select the components with component names that match the query. Search results may appear outside of the visible area, click *Fit to Screen* from the lower-left toolbar to resize the *Topology* to show all components.